packages/shared/lib/sdk/sync.ts

```diff
@@ -417,9 +412,24 @@ export class NangoAction {
         return (metadata['fieldMapping'] as Metadata) || {};
     }
 
-    public async log(content: string, userDefinedLevel?: UserLogParameters): Promise<void> {
+    /**
+     * Log
+     * @desc Log a message to the activity log which shows up in the Nango Dashboard
+     * note that the last argument can be an object with a level property to specify the log level
+     * example: await nango.log('This is a log message', { level: 'error' })
+     */
+    public async log(...args: any[]): Promise<void> {
+        const lastArg = args[args.length - 1];
```

Comment:
what happens if log is called without arguments?

Reply:
Error throw with this update: https://github.com/NangoHQ/nango/pull/1601/commits/ceb892eaaedb1c680a2f2332cd9325fa1999b60d#diff-f52f0d61cd6802499f5a31aca89d3aec3df278a751065390c02b6bbdb24c0840R427

Reply:
`console.log` prints an empty string when passing no argument. Should we do the same and have log to be a no op when no arg is passed

Reply:
Sure, updated with https://github.com/NangoHQ/nango/pull/1601/commits/b0734a23f5571cfccebbf9bc225bdb2adf487c86